app/src/main/res/values-es/strings.xml

```diff
@@ -390,6 +390,17 @@
     <string name="settingsAiEngineDescription">Seleccione su asistente de redacción</string>
     <string name="settingsAppLock">Bloqueo de aplicaciones</string>
     <string name="settingsAuthorizeTracking">Autorizar el seguimiento</string>
+    <string name="settingsAutoAdvanceDescription">Seleccione qué mostrar cuando se archiva o elimina un mensaje</string>
+    <string name="settingsAutoAdvanceFollowingThreadDescription">Mostrar el siguiente hilo al archivar o borrar\n</string>
+    <string name="settingsAutoAdvanceFollowingThreadTitle">Siguiendo el hilo</string>
+    <string name="settingsAutoAdvanceListOfThreadsDescription">Mostrar lista de hilos al archivar o borrar</string>
+    <string name="settingsAutoAdvanceListOfThreadsTitle">Lista de temas</string>
```

Comment:
The term "conversation" is usually not translated as "temas" in spanish translations, maybe check if you can find synonyms that are already used in other already existing translations like "conversación"